src/Azure.Functions.Cli/Actions/AzureActions/BaseAzureAction.cs

```diff
@@ -16,15 +14,6 @@ public BaseAzureAction(IArmManager armManager)
         }
 
         public async Task Initialize()
-        {
-            try
-            {
-                await _armManager.Initialize();
```

Comment:
Under the hood, this calls `ArmManager.SelectTenantAsync`, which calls `_authHelper.GetToken` and `_authHelper.AcquireTokens()` which I don't think makes much sense if there is a `login` command. this should also resolve #112